changelogs/3.0.0.md

```diff
@@ -0,0 +1,478 @@
+  information on memory consumed by data (gh-6762).
+* Introduced the new built-in system event `box.wal_error` that is broadcast
+  whenever Tarantool fails to commit a transaction to the write-ahead log
+  (gh-9405).
+* Introduced transaction-related events `box.before_commit`, `box.on_commit`,
+  and `box.on_rollback` for the new trigger registry. One of the main advantages
+  of the new triggers is that they can be set for all transactions rather than
+  setting them within each transaction (gh-5717, gh-8656).
+
+### Memtx
+
+* Added memtx read view statistics to `box.stat.memtx()` (gh-8501).
+
+### Vinyl
+
+* Introduced the `memory.tuple` statistic for `box.stat.vinyl()` that shows
+  the total size of memory occupied by all tuples allocated by the Vinyl engine
+  (gh-8485).
+
+### Replication
+
+* Added a new `bootstrap_leader` configuration option to specify the node from
+  which a replica should bootstrap. To do this, set `box.cfg.bootstrap_strategy`
+  to `'config'` and set `bootstrap_leader` value to either the URI or UUID of
+  the desired bootstrap leader. For example:
+  ```lua
+    box.cfg{
+        bootstrap_strategy = 'config',
+        bootstrap_leader = 'localhost:3301',
+        replication = {
+            'localhost:3301',
+            'localhost:3302',
+        },
+        listen = '3302',
+    }
+  ```
+  (gh-7999).
+* You may now control which node new replicas choose as a bootstrap leader
+  without touching node config. To do so, set `box.cfg.bootstrap_strategy` to
+  `'supervised'`, and the nodes will only bootstrap off the node on which you
+  called `box.ctl.make_bootstrap_leader()` last.
+  This works on an empty replica set bootstrap as well: start the admin console
+  before configuring the nodes. Then configure the nodes:
+  ```lua
+  box.cfg{
+      bootstrap_strategy = 'supervised',
+      replication = ...,
+      listen = ...,
+  }
+  ```
+  Finally, call `box.ctl.make_bootstrap_leader()` through the admin console
+  on the node you want to promote. All the nodes will bootstrap off that node
+  (gh-8509).
+* A new option `box.cfg.cluster_name` allows assigning a human-readable name to
+  the entire cluster. It has to match in all instances and is displayed in
+  `box.info.cluster.name` (gh-5029).
+
+* A new option `box.cfg.replicaset_name` allows assigning a human-readable name
+  to the replicaset. It works the same as `box.cfg.replicaset_uuid`. Its value
+  must be the same across all instances of one replicaset. The replicaset name
+  is displayed in `box.info.replicaset.name` (gh-5029).
+
+* A new option `box.cfg.instance_name` allows assigning a human-readable name to
+  the instance. It works the same as `box.cfg.instance_uuid`. Its value must be
+  unique in the replicaset. The instance name is displayed in `box.info.name`.
+  Names of other replicas in the same replicaset are visible in
+  `box.info.replication[id].name` (gh-5029).
```

Comment:
```suggestion
* A new option `box.cfg.replicaset_name` allows assigning a human-readable name
  to the replica set. It works the same as `box.cfg.replicaset_uuid`. Its value
  must be the same across all instances of one replica set. The replica set name
  is displayed in `box.info.replicaset.name` (gh-5029).

* A new option `box.cfg.instance_name` allows assigning a human-readable name to
  the instance. It works the same as `box.cfg.instance_uuid`. Its value must be
  unique in the replicaset. The instance name is displayed in `box.info.name`.
  Names of other replicas in the same replica set are visible in
  `box.info.replication[id].name` (gh-5029).
```

Reply:
updated (you missed one 4th line from bottom), force pushed